target/linux/ath79/dts/qca9563_dlink_dir-842-c1.dts

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
```

Comment:
These includes are already in the dtsi...

Reply:
Removed, thanks!